Atomic/sign.py

```diff
@@ -88,7 +88,7 @@ def make_sig_dirs(sig_path):
             # TODO
             # perhaps revisit directory permissions
             # when complete use-cases are known
-            os.makedirs(sig_path, '0777')
+            os.makedirs(sig_path, mode=0o777)
```

Comment:
Don't specify it and it should work in both formats.  755 is probably what you want the permissions to be.